test/evergreen.yml

```diff
@@ -1735,3 +1735,17 @@ buildvariants:
   - name: generate-datafile-big-endian
   - name: verify-datafile-big-endian
   - name: verify-datafile-from-little-endian
+
+- name: ubuntu1804-powerpc
+  display_name: Ubuntu 18.04 PowerPC
```

Comment:
I'd prefer using the well-known alias name "ppc/PPC" :)

Reply:
Updated the task and display name to ppc/PPC .